download-basket/src/main/java/downloadbasket/helpers/Helpers.java

```diff
@@ -0,0 +1,52 @@
+package downloadbasket.helpers;
+
+import fi.nls.oskari.log.LogFactory;
+import fi.nls.oskari.log.Logger;
+
+public class Helpers {
+	
+	private static final Logger log = LogFactory.getLogger(Helpers.class);
+	
+	/**
+	 * Get GetFeatureInfoUrl for proxy
+	 * @param url a url
+	 * @param projection a projection
+	 * @param bbox a bbox
+	 * @param width a width
+	 * @param height a heigth
+	 * @param x a x
+	 * @param y a y
+	 * @param layerName a layer name
+	 * @return getFeatureInfoUrl
+	 */
+	public static String getGetFeatureInfoUrlForProxy(String url, String projection, String bbox, String width, String height, String x, String y, String layerName) {
+		String wmsUrl = url+"?SERVICE=WMS&VERSION=1.1.1&REQUEST=GetFeatureInfo&SRS="+projection
```

Comment:
Check IOHelper.constructUrl() for properly concatenating parameters to a base url